src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -217,7 +218,7 @@ private async Task InitializeAsync()
             _solutionEvents.ProjectRemoved += OnEnvDTEProjectRemoved;
             _solutionEvents.ProjectRenamed += OnEnvDTEProjectRenamed;
 
-            var vSStd97CmdIDGUID = VSConstants.GUID_VSStandardCommandSet97.ToString("B");
+            var vSStd97CmdIDGUID = VSConstants.GUID_VSStandardCommandSet97.ToString("B", CultureInfo.CurrentCulture);
```

Comment:
I think the guid changes are incorrect. 
My bad, I should have caught that.

In particular, GUIDS should not be formatted different as it's based on what the rest of VS uses, which is the guid version.


Now the *good* part is that to my knowledge, guids do not support culture formatting, but that doesn't mean we shouldn't fix this..